python/GangaLHCb/Lib/GaudiRunApp/GaudiRunAppHandlers.py

```diff
@@ -192,6 +193,9 @@ def prepare(self, app, appconfig, appmasterconfig, jobmasterconfig):
             raise ApplicationConfigurationError(None, 'Prepared folders not supported yet, please fix this in future')
         else:
             for f in input_files:
```

Comment:
`I think the changes highlighted here have been implemented in the GaudiExec class.`
The GaudiExec class now in develop is the new place where the LHCb Cmake support is being developed.